### PR TITLE
Added Swift Package Manager Support 

### PR DESCRIPTION
**Pull request by karthik-kumar-mm**
- Segregated the test files from source files.  
- Added swift manifest file and updated the source files path and test files path.
- In development app removed the old test file reference and added from new path. 
- Updated the ReadMe file  - How to install via Swift Package Manager

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+let package = Package(
+    name: "BarricadeKit",
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "BarricadeKit",
+            targets: ["BarricadeKit"]),
+    ],
+    dependencies: [
```
Comment (pavan309) [inline Package.swift]: The empty array of dependencies and the basic comments in the file can be removed.

**Comment by johntmcintosh** [comment]
Thanks @karthik-kumar-mm ! At first glance this looks good, but I'd like to pull down and build once before merging and then will plan to tag a new release after I do. I have a busy next few days, but will as soon as I can.

**Comment by johntmcintosh** [comment]
@karthik-kumar-mm just got a chance to take a look, thanks again!

**Comment by johntmcintosh** [comment]
@karthik-kumar-mm I just pushed up a 2.0.5 tag that you should be able to use to point to this

**Pull request merged**